### ISSUE TITLE
like by feed not working, latest github release 

### ISSUE THREAD

**Issue by gwens0n** [InstaPy/InstaPy#6297]
## Expected Behavior
like by feed working 
## Current Behavior
like by feed sends 0/24 and ends program
## Possible Solution (optional)
not sure 
## InstaPy configuration

from instapy import InstaPy
from instapy import smart_run
session = InstaPy(username="soind", password="", headless_browser=False)
with smart_run(session):

    session.set_skip_users(skip_no_profile_pic=True)
    session.set_do_like(enabled=True, percentage=100)
<img width="703" alt="Unbenannt" src="https://user-images.githubusercontent.com/89553674/130873595-5d8a5304-f94f-4dac-91ff-503a4840b5df.png">

**Comment by ErezAmihud** [comment]
I remember it happened to me, I think https://github.com/timgrossmann/InstaPy/pull/6143/files fixed it

**Comment by gdr2404** [comment]
> I remember it happened to me, I think https://github.com/timgrossmann/InstaPy/pull/6143/files fixed it

I tried that, unfortunately it didn't work for me.

**Comment by ErezAmihud** [comment]
Then this will
`pip install -I https://github.com/schealex/InstaPy/zipball/develop`

**Comment by lolicarlos** [comment]
> Then this will
> `pip install -I https://github.com/schealex/InstaPy/zipball/develop`

still cant like my feeds
didnt work for me .

**Comment by laboratoriodosfundos** [comment]
I started getting the same error, has anyone been able to solve this?

**Comment by gdr2404** [comment]
yeah me too, this is the result i'm getting, video starts about 45 seconds in. sorry for the delay, im recording through a vnc and the whole operation seems slow also, though the system is a i9

https://youtu.be/X3bttxuiLRo

**Comment by ErezAmihud** [comment]
@lolicarlos @gdr2404 @laboratoriodosfundos 

A long ahot but I didn't see anyone else replying and it worked on my machine

This could be connected to the `hide selenium extension: error`. Can any of you try to follow these instructions: https://github.com/ErezAmihud/InstaPy/blob/feature/force-firefox-esr/docs/before-installing.md

And then run the bot again (make sure you have the version in master with `pip install -I https://github.com/timgrossmann/InstaPy/zipball/master`

**Comment by gdr2404** [comment]
I tried that, It now reads... hide selenium extension: ok with the earlier version of FF but its still not liking any posts.

**Comment by laboratoriodosfundos** [comment]
It looks like Instagram has changed a feed structure.
I think someone should update Instapy.
